src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+
+    public RightTriangle(int size, String color) {
+        super(color);
+        this.side = size;
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.pow(getSide(), 2) * (Math.sqrt(3) / 4);
```

Comment:
The formula used here is for the area of an equilateral triangle, not a right triangle. For a right triangle with equal legs, the area should be calculated as (side * side) / 2. Please revise this calculation.